web/looking-good/src/utils/theme.js

```diff
@@ -5,9 +5,11 @@ import { pink } from '@material-ui/core/colors'
 const theme = createMuiTheme({
   palette: {
     primary: {
-      main: '#4300FF',
+      //main: '#4300FF',
+      main: '#2839aa',
```

Comment:
The colors we were using came from the Can't Stop Columbus Branding document. We probably shouldn't change them unless that document has been updated

Reply:
Yeah so Rachyl told me, have swapped back to the original colors.